ci/infra/openstack/variables.tf

```diff
@@ -116,3 +116,13 @@ variable "rmt_server_name" {
   default     = ""
   description = "SUSE Repository Mirroring Server Name"
 }
+
+variable "cpi_enable" {
+  default     = false
```

Comment:
Do you think we need to modify the examples file as well or not needed?

Reply:
I think it is not needed, CPI will be enabled only in specific use cases